rabies/analysis_pkg/cpca/modeling.py

```
@@ -0,0 +1,203 @@
+        if lim < tol:
+            if verbose > 1:
+                print(str(i)+' iterations to converge.')
+            break
+        if i == max_iter-1:
+            if verbose > 0:
+                print(
+                    'Convergence failed. Consider increasing max_iter or decreasing tol.')                
+    
+    return Cs,Ws,C_
+
+def cpca(X, C_prior, sequence = ['s','s','s','s'], verbose=False):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)